Enterprise_Dbcache_WpdbInjection_Cluster.php

```diff
@@ -874,9 +876,9 @@ function _db_connect_fallback() {
 		if ( !mysqli_select_db( $this->wpdb_mixin->dbh, DB_NAME ) )
 			return $this->wpdb_mixin->bail( "We were unable to select the database." );
 		if ( !empty( $this->wpdb_mixin->charset ) ) {
-			$collation_query = "SET NAMES '$this->wpdb_mixin->charset'";
+			$collation_query = "SET NAMES '{$this->wpdb_mixin->charset}'";
```

Comment:
This is the correct way to handle object variables in a string. This will throw a PHP error otherwise.